src/etc/inc/auth.inc

```diff
@@ -474,6 +474,8 @@ function local_user_set(& $user) {
 		$user_shell = "/bin/tcsh";
 	} elseif (userHasPrivilege($user, "user-copy-files")) {
 		$user_shell = "/usr/local/bin/scponly";
+	} elseif (userHasPrivilege($user, "user-copy-files-chroot")) {
```

Comment:
Well, I don't care either way, certainly can be done... I've been thinking about adding some input validations there for the "conflicts", but that got nowhere since I couldn't figure out where it should be put at all.